crates/notary/client/src/client.rs

```diff
@@ -113,6 +113,9 @@ pub struct NotaryClient {
     /// Port of the notary server endpoint.
     #[builder(default = "self.default_port()")]
     port: u16,
+    /// URL path of the notary server endpoint, e.g. version.
```

Comment:
```suggestion
    /// URL path prefix of the notary server endpoint, e.g. "https://<host>:<port>/<path_prefix>/...".
```

Reply:
Makes sense , that is a lot more undestandable